packages/components/app/styles/components/table.scss

```diff
@@ -32,16 +32,18 @@
     text-align: left;
     background-color: var(--token-color-surface-strong);
 
-    &.hds-table__th--sortable {
+    &.hds-table__th-sort {
       padding: 0; // we're taking it out here because we need it on the interactive content instead
     }
 
-    a { // will refactor this to a button. Use Hds::Button?
+    button { // will refactor this to a button. Use Hds::Button?
```

Comment:
Nit: Delete outdated comment? `// will refactor this to a button. Use Hds::Button?`